addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -821,6 +830,22 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Generates new ids for the cloned elements
+     * 
+     * @override
+     */
+    onClone() {
+        const clonedEl = this.$target[0];
+        const newId = this._generateUniqueID();
+        for (const el of clonedEl.querySelectorAll('[id] :not(div)')) {
+            el.id = newId;
+        }
+        for (const el of clonedEl.querySelectorAll('[for]')) {
+            el.setAttribute('for', newId);
+        }
+        this._renderList();
```

Comment:
Need to rebase, this doesnt exists anymore, see 9304f8fa7bb94f21dd4caa83f9679ffd1642691b

Reply:
Actually the module itself doesnt exists anymore in master, but everything is gonna be fine :D